operators/src/source/ogr_source.rs

```diff
@@ -1190,9 +1256,13 @@ mod tests {
                     step: 42,
                 }),
             },
+            default_geometry: Some(TypedGeometry::MultiPoint(
```

Comment:
I guess we need one test here for using the default geometry, right?

Reply:
dae2a60b1eb1c7c9fd50d3676daaececcfda0b79